src/inspector_io.cc

```diff
@@ -236,6 +236,7 @@ void InspectorIo::WriteCbIO(uv_async_t* async) {
 template<typename Transport>
 void InspectorIo::WorkerRunIO() {
   uv_loop_t loop;
+  loop.data = nullptr;
```

Comment:
According to [libuv docs](http://docs.libuv.org/en/v1.x/loop.html#c.uv_loop_t.data), this should be initialized in the next line with `uv_loop_init()`: 

> <dl><dt>void* uv_loop_t.data</dt><dd>Space for user-defined arbitrary data. libuv does not use this field. libuv does, however, initialize it to NULL in <code>uv_loop_init()</code>, and it poisons the value (on debug builds) on <code>uv_loop_close()</code>.</dd></dl>